lib/widget/helpers/controllers.dart

```diff
@@ -239,6 +260,92 @@ class BoxController extends WidgetController {
     return setters;
   }
 
+  void parseBorderColor(value) {
+    final parsedValue = Utils.optionalString(value);
+
+    if (parsedValue == null) return;
+
+    final individualBorderColorData = parsedValue.trim().split(' ');
+
+    if (individualBorderColorData.length == 1) {
+      borderColor = convertStringToColor(individualBorderColorData[0]);
+
+      topBorderColor = borderColor;
+      bottomBorderColor = borderColor;
+      leftBorderColor = borderColor;
+      rightBorderColor = borderColor;
+    } else if (individualBorderColorData.length == 2) {
+      topBorderColor = convertStringToColor(individualBorderColorData[0]);
```

Comment:
shorten and don't repeat logic to minimize typo errors:
topBorderColor = bottomBorderColor = ..
Same for below.